src/utils/fetchClient.ts

```diff
@@ -0,0 +1,26 @@
+const BASE_URL = 'https://fe-oct23-code-fairies-backend.onrender.com/';
```

Comment:
That reminded me. Uhh, install `dotenv` package, set it up in this file using `dotenv.config()` and use the base url value from `.env`